src/content/learn/editor-setup.md

```diff
@@ -1,62 +1,62 @@
 ---
-title: Editor Setup
+title: Persiapan Editor
 ---
 
 <Intro>
 
-A properly configured editor can make code clearer to read and faster to write. It can even help you catch bugs as you write them! If this is your first time setting up an editor or you're looking to tune up your current editor, we have a few recommendations.
+Sebuah editor yang terkonfigurasi dengan baik dapat membuat kode lebih mudah dibaca dan lebih cepat ditulis. Ini bahkan dapat membantu Anda menangkap *bug* saat Anda menulisnya! Jika ini adalah pertama kalinya Anda mempersiapkan editor atau Anda ingin menyetel ulang editor Anda saat ini, kami memiliki beberapa rekomendasi.
 
 </Intro>
 
 <YouWillLearn>
 
-* What the most popular editors are
-* How to format your code automatically
+* Apa saja editor yang paling populer
+* Cara memformat kode Anda secara otomatis
 
 </YouWillLearn>
 
-## Your editor {/*your-editor*/}
+## Editor Anda {/*your-editor*/}
```

Comment:
Ini slug nya perlu diganti juga gak sih?

Reply:
according to the [guidelines](https://github.com/reactjs/id.react.dev/blob/main/TRANSLATION.md), translating the Heading ID will ruin the navigation